data/map systems.txt

```diff
@@ -31410,6 +33275,70 @@ system Tarazed
 			distance 228
 			period 16.9693
 
+system "Tau Ceti"
+	pos -168.19488 -53.953889
+	government Syndicate
+	attributes core
+	link "Epsilon Indi"
+	link Menkar
+	link Zaurak
+	asteroids "small rock" 30 5.0232
+	asteroids "medium rock" 34 5.2052
+	asteroids "large rock" 11 1.82
+	asteroids "small metal" 4 4.6228
+	asteroids "medium metal" 199 2.4024
+	asteroids "large metal" 46 4.4044
+	minables iron 61 3.64072
+	haze _menu/haze-133
+	fleet "Small Core Merchants" 800
+	fleet "Large Core Merchants" 1000
+	fleet "Small Syndicate" 1600
+	fleet "Large Syndicate" 2500
+	fleet "Small Core Pirates" 2000
+	fleet "Large Core Pirates" 4000
+	fleet "Human Miners" 2000
+	habitable 861.25
```

Comment:
The lone A8 star in Elo Chigo has a habitable range of 3000, this should be slightly higher because of the second star.